btr-web/btr-layouts/layouts/business.vue

```diff
@@ -3,8 +3,27 @@
     <BcrosHeader />
     <BcrosBreadcrumb v-if="crumbConstructors.length > 0" :crumb-constructors="crumbConstructors" />
     <BcrosBusinessDetails />
-    <div class="mx-auto px-4 w-full max-w-[1360px]">
-      <slot />
+    <div class="mx-auto p-4 w-full max-w-bcroslg flex justify-center items-start">
+      <div class="max-w-bcrosmd p-4">
+        <slot />
+      </div>
+      <div class="hidden bcroslg:block p-4">
+        <BcrosWidgetsFee :fees="payFeesWidget.fees" data-cy="pay-fees-widget">
```

Comment:
This should be sticky / move with the screen when you scroll down.

This is an example: https://dev.search.business.bcregistry.gov.bc.ca/open/CP1002539